### PR TITLE
Created and updated docs for 1.21-14

### PR DESCRIPTION
**Pull request by kschumy**
Created and updated docs for 1.21-14

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/releases/1-21/14/CHANGELOG-v1-21-eks-14.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* **0024-EKS-PATCH-Fix-kubectl-version-unit-test.patch**
+  * Cherry-pick of upstream Kubernetes PR
+    [#103955](https://github.com/kubernetes/kubernetes/pull/103955), which is
+    included in Kubernetes 1.23.
+  * Fixes configuration requirement that results in test 
+    TestNewCmdVersionWithoutConfigFile failing for some developers.
+
+### Patches Removed
+* **0019-EKS-PATCH-AWS-Set-max-results-if-its-not-set.patch**
+  * The Updated version of Kubernetes cherrypicked this change in PR
```
Comment (kschumy) [inline docs/contents/releases/1-21/14/CHANGELOG-v1-21-eks-14.md:38]:
```suggestion
  * The updated version of Kubernetes cherrypicked this change in PR
```

File: docs/contents/index.md [hunk 2]
```diff
@@ -128,6 +128,7 @@ if you are interested in the AL2 container runtime kernel version.
 * [v1-22-eks-1](releases/1-22/1/index.md) (March 6, 2022)
 
 #### EKS-D 1.21 Version Dependencies
+* [v1-21-eks-14](releases/1-21/14/index.md) (May 18, 2022)
```
Comment (kschumy) [inline docs/contents/index.md:131]:
```suggestion
* [v1-21-eks-14](releases/1-21/14/index.md) (May 17, 2022)
```

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**